### PR TITLE
Op 774 laboratory honoring date and time now

### PR DESCRIPTION
**Pull request by mwithi**
See OP-774.

Paired with https://github.com/informatici/openhospital-gui/pull/1138.

Will follow OP-781 after v1.11.3

### CODE REVIEW

File: src/main/java/org/isf/lab/model/Laboratory.java [hunk 1]
```diff
@@ -124,7 +125,7 @@ public Laboratory() { }
 	public Laboratory(Exam aExam,GregorianCalendar aDate,String aResult,
 			String aNote, Patient aPatId, String aPatName){
 		exam=aExam;
-		registrationDate=aDate;
+		labDate=aDate;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/lab/model/Laboratory.java:128]: Nitpick: the formatting in this and the next method could use some work.

File: src/main/java/org/isf/lab/model/Laboratory.java [hunk 2]
```diff
@@ -165,14 +166,22 @@ public void setExam(Exam aExam){
...
+	/**
+	 * the field has been replaced by <code>createdDate()</code> and it's not meant to be managed by the user (Spring managed)
+	 */
+	@Deprecated
 	public void setExamDate(GregorianCalendar exDate) {
 		this.examDate = exDate;
 	}	
 	public void setDate(GregorianCalendar aDate){
-		registrationDate=aDate;
+		labDate=aDate;
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/lab/model/Laboratory.java:184]: again more formatting here and the next routine

**Review by dbmalkovsky — APPROVED** [review]

File: src/main/java/org/isf/lab/model/Laboratory.java (hunk 1 above)
Comment (mwithi) [reply src/main/java/org/isf/lab/model/Laboratory.java]: Fixed, thanks!

File: src/main/java/org/isf/lab/model/Laboratory.java (hunk 2 above)
Comment (mwithi) [reply src/main/java/org/isf/lab/model/Laboratory.java]: Fixed, thanks!

**Pull request merged**